docs/feature_request.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+
+title: "REQUEST: "
+labels:  feature-request "P3 - Normal"
+assignees: kreyren nntoan
```

Comment:
Having all issues assigned to ya'll may get a bit spamming in your inbox. I'd personally recommend against doing this.
``` suggestion
```

Reply:
This PR was made by the previous maintainers (they are still maintainers technically, but they became inactive), but I don't have a plan to merge this as is. Maybe I'll pick some parts from this PR someday when I create the templates, but I now don't see the necessity of the templates.